README.md

```diff
@@ -173,6 +173,23 @@ Your email will be acknowledged within 24 hours, and you’ll receive a more
 detailed response to your email within 48 hours indicating the next steps in
 handling your report.
 
+There is not any hard and fast rule to determine if a bug is worth reporting as
+a security issue. The general rule is any issue worth reporting via the
+security address must allow an attacker to affect the confidentiality, integrity
```

Comment:
I think it may be worth defining "attacker" here. What abilities does the attacker have? (e.g., do they interface with the system over HTTP, fs, etc.?)